config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  scope "(:locale)", locale: /en|vi/ do
+    resources :static_pages
```

Comment:
khi khai báo như thế này thì sẽ sinh ra những gì?

Reply:
> khi khai báo như thế này thì sẽ sinh ra những gì?

Khi khai báo như vậy thì chỉ sinh ra trang local:host Ruby on Rails mà không dẫn đến các trang html mà t đã tạo

Reply:
> khi khai báo như thế này thì sẽ sinh ra những gì?

Khi khai báo như này thì sẽ sinh ra các đường dẫn tùy chọn của local: để hiển thị ngôn ngữ chuyển đổi 
Ví dụ như local:host/en để ngôn ngữ hiển thị văn bản tiếng anh
hay local:host/vi để hiển thị ngôn ngữ văn bản tiếng việt

Reply:
chị hỏi dòng 
```suggestion
    resources :static_pages
```
cơ mà

Reply:
> chị hỏi dòng
> 
> cơ mà

Dòng lệnh đó sẽ sử dụng các trang trong folder static_pages, chứ không phải từ các resources khác.

Reply:
Dòng lệnh này sinh ra đầy đủ tất cả các action và _path của static_page ạ

Reply:
cụ thể hơn xem nào?

Reply:
các action: index, create, show, edit, destroy, new, update và các _path được sinh ra tương ứng với mỗi action ạ

Reply:
vậy trong static_pages em có các action đó không?

Reply:
trong static_pages của em thì chưa có cái đó ạ nhưng khi check rails router thì nó hiện ra như vậy ạ

Reply:
vậy em có cần khai báo resources ở đây k?

Reply:
vâng để em sửa